src/rust/src/dataframe/io.rs

```diff
@@ -38,4 +39,95 @@ impl PlRDataFrame {
             .map_err(RPolarsErr::from)?;
         Ok(df.into())
     }
+
+    #[cfg(not(target_arch = "wasm32"))]
+    pub fn write_parquet(
```

Comment:
As I mentioned somewhere before, I am wondering if it is worthwhile to add this now, given that we have problems with this (pola-rs/polars#13040)

Reply:
I think this is an important function. There's no indication that the linked issue will be resolved any time soon so I don't think we should wait for it.

Also `write_parquet()` is already supported in `r-polars`.

Reply:
Sorry, I was confused. I think the above issue is related to reading and there are two write functions, which is not a problem.